packages/create-amplify/src/amplify_project_creator.ts

```diff
@@ -31,51 +32,55 @@ export class AmplifyProjectCreator {
     private readonly tsConfigInitializer: TsConfigInitializer,
     private readonly gitIgnoreInitializer: GitIgnoreInitializer,
     private readonly projectRoot: string,
-    private readonly logger: typeof console = console
+    private readonly logger: Logger
   ) {}
 
   /**
    * Executes the create-amplify workflow
    */
   create = async (): Promise<void> => {
-    this.logger.log(`Validating current state of target directory...`);
+    await this.logger.debug(`Validating current state of target directory...`);
```

Comment:
Can we make logger `sync` ? My quick search shows the log is marked async for no reason.

Reply:
Done in [fb89cac](https://github.com/aws-amplify/amplify-backend/pull/595/commits/fb89cacbc4748373aa55231303d37aa8d05dccbc)